File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lst = []
        for i in range(len(nums)):
            for k in range(i+1, len(nums)):
                save = nums[i] + nums[k]
                if save == target:
                    lst.append(i)
                    lst.append(k)
                    return lst                    

File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
    hashMap := make(map[int]int)
    for idx, val := range nums {
        hashMap[val] = idx
    }

    for idx, val := range nums {
        diff := target - val
        secondIdx, exist := hashMap[diff]
        if idx != secondIdx && exist {
            return []int{idx, secondIdx}
        }
    }

    return []int{0, 0}
}